packages/example-getting-started/README.md

```diff
@@ -2,23 +2,76 @@
 
 This is the basic tutorial for getting started with Loopback 4!
 
-To get started, jump into the
-[Prerequisites and setup](docs/1-prerequisites-and-setup.md) section.
+## Overview
 
-## Tutorial Steps
+This tutorial demonstrates how to create a basic API for a todo list using
+LoopBack 4.
 
-1. [Prerequisites and setup](docs/1-prerequisites-and-setup.md)
-1. [Scaffolding your application](docs/2-scaffold-app.md)
-1. [Adding the legacy juggler](docs/3-add-legacy-juggler.md)
-1. [Add your Todo model](docs/4-todo-model.md)
-1. [Add a datasource](docs/5-datasource.md)
-1. [Add a repository](docs/6-repository.md)
-1. [Add a controller](docs/7-controller.md)
-1. [Putting it all together](docs/8-putting-it-together.md)
+## Setup
+
+You'll need to make sure you have some things installed:
+- [Node.js](https://nodejs.org/en/) at v8.x or greater
+
+Additionally, this tutorial assumes that you are comfortable with
+certain technologies, languages and concepts.
+- JavaScript (ES6)
```

Comment:
Should this be JavaScript or **TypeScript**?

Reply:
JavaScript.

You can get away with not knowing TypeScript for this tutorial, since we give you complete working examples. Not knowing JavaScript could be a bit more problematic for consuming this tutorial.